packages/svelte/src/compiler/phases/3-transform/client/utils.js

```diff
@@ -292,7 +292,17 @@ export function serialize_set_binding(node, context, fallback, options) {
 
 	const serialize = () => {
 		if (left === node.left) {
-			if (binding.kind === 'prop' || binding.kind === 'bindable_prop') {
+			const initial_bindable_proxy =
```

Comment:
It's impossible to have a default value that is one of the things you guard against, so I think it's fine to just type cast here instead